### PR TITLE
[RFC] Qt/Wx: Turn Qt into the default interface

### PR DESCRIPTION
**Pull request by spycrab**
Also see the [Qt feature report](https://gist.github.com/spycrab/84b9f5c2b182455495db84973433dc84).

### CODE REVIEW

**Comment by MayImilae** [comment]
When are you planning to start pushing this for merging?

**Comment by spycrab** [comment]
When Qt is feature-complete (so approximately this summer).

**Comment by spycrab** [comment]
When Qt is feature-complete (so approximately this summer).

**Comment by spycrab** [comment]
This PR breaks the windows buildbots atm, closing it for now.

File: Source/Core/DolphinQt2/CMakeLists.txt [hunk 1]
```diff
@@ -113,28 +113,30 @@ add_executable(dolphin-emu-qt2
...
 
-target_link_libraries(dolphin-emu-qt2
+target_link_libraries(dolphin-emu
 PRIVATE
   core
   Qt5::Widgets
   uicommon
 )
 
+set(DOLPHINQT2_BINARY dolphin-emu)
```
Comment (BhaaLseN) [inline Source/Core/DolphinQt2/CMakeLists.txt]: This doesn't seem to be used; is this necessary or can it be removed?

**Comment by MayImilae** [comment]
When #6650 and #6659 are merged, I think it's time to do this!

**Comment by autofire372** [comment]
This still isn't building for Windows...

**Comment by spycrab** [comment]
It does. The buildbot just needs to do a clean rebuild once.

**Pull request merged**